src/pages/artists.be.jsx

```diff
@@ -1,11 +1,11 @@
-/* eslint-disable react/prop-types */
 import React from 'react';
 import { graphql, Link } from 'gatsby';
-// import PropTypes from 'prop-types';
+import PropTypes from 'prop-types';
+
 import Layout from '../components/layout/Layout';
 
 const Item = queryData => (
```

Comment:
change variable name. Item is not correct.

Reply:
Done 061371c